python/lsst/obs/base/cameraMapper.py

```diff
@@ -1060,6 +1063,8 @@ def _resolveFilters(definitions, idFilter, filterLabel):
             matches.intersection_update(definitions.findAll(filterLabel.bandLabel))
         return matches
 
+    @deprecated(reason="Removed with no replacement (due to end of gen2). Will be removed after v23.",
+                category=FutureWarning)
     def _setFilter(self, mapping, item, dataId):
         """Set the filter information in an Exposure.
```

Comment:
I'm not sure I agree with this method being deprecated -- I'm using it to patch the `FilterLabel` created by reading old files, since `afw` can't access the `FilterDefinitions`. I think it should stick around until the end of Gen 2. (I see you didn't remove it from `_standardizeExposure`, which I 👍.)